### PR TITLE
Define auth endpoints as not requiring auth.

### PR DESCRIPTION
**Pull request by cbookg**
Also format the openapi yaml

### CODE REVIEW

**Comment by cbookg** [comment]
Is my yaml formatter inline with others?

**Comment by aaron-wang** [comment]
> Is my yaml formatter inline with others?

Yes. Although, my formatter maintains the padding within square brackets. Otherwise, everything else looks good.
So,
```yaml
authorization: [ openiAd, email, profile ]
```
And not:
```yaml
authorization: [openiAd, email, profile]
```

@rogerwangcs: Does your formatter work the same?

File: service/src/main/resources/api/openapi.yml [hunk 1]
```diff
@@ -40,20 +40,59 @@ paths:
...
+        "403":
+          $ref: "#/components/responses/PermissionDenied"
+        "500":
+          $ref: "#/components/responses/ServerError"
+
+  /auth/refreshToken:
+    parameters:
+      - $ref: "#/components/parameters/RefreshToken"
+    get:
+      security: []
```
Comment (yuhuyoyo) [inline service/src/main/resources/api/openapi.yml:67]: it's hard to see with the format. you just added this?

Comment (cbookg) [reply service/src/main/resources/api/openapi.yml:67]: and removed the "api" prefix from the paths

**Review by yuhuyoyo — APPROVED** [review]

File: service/src/main/resources/api/openapi.yml (hunk 1 above)
Comment (yuhuyoyo) [reply service/src/main/resources/api/openapi.yml:67]: ah duh.

**Review by aaron-wang — APPROVED** [review]

**Pull request merged**